kubernetes/kube-state-metrics-deployment.yaml

```diff
@@ -53,7 +53,7 @@ spec:
           - --container=kube-state-metrics
           - --cpu=100m
           - --extra-cpu=1m
-          - --memory=30Mi
+          - --memory=200Mi
           - --extra-memory=2Mi
-          - --threshold=5
-          - --deployment=kube-state-metrics
\ No newline at end of file
+          - --threshold=100
```

Comment:
The docs of the addon-resizer say this about the threshhold flag

> A number between 0-100. The dependent's resources are rewritten when they deviate from expected by more than threshold.

I'm not even sure what the numbers mean, but given it's 0-100 I'm thinking it's %, which would mean that only once the memory would accumulate to 400Mi it would trigger. Am I understanding this correctly, or does it mean when 100 x extra memory is reached start modifying?